### PR TITLE
Add release documentation involving Github Actions

### PR DESCRIPTION
**Pull request by swillner**
Very minor PR but I thought I commit directly to master too much ;)

### CODE REVIEW

File: docs/development.rst [hunk 1]
```diff
@@ -230,6 +230,23 @@ there).
...
+
+Creating a release
+==================
+
+OpenSCM uses designated Github Actions to upload the package to PyPI
+(and, in the future, also to Conda). To create a release:
+
+1. Change the "master" header in ``CHANGELOG.rst`` to the release
+   version number (not starting with "v", e.g. "1.2.3") and create a
+   new, empyy "master" header above. Commit these changes.
```
Comment (znicholls) [inline docs/development.rst]:
```suggestion
   new, empty "master" header above. Commit these changes.
```

Comment (znicholls) [inline docs/development.rst]: 'Commit these changes.' --> 'Commit these changes with the message, 'Prepare for release of vVERSIONNUMBER'' e.g. 'Prepare for release of v1.2.3'

**Comment by znicholls** [comment]
> Very minor PR but I thought I commit directly to master too much ;)

We shouldn't be able to commit directly to master at all?

File: docs/development.rst [hunk 2]
```diff
@@ -230,6 +230,25 @@ there).
...
+Creating a release
+==================
+
+OpenSCM uses designated Github Actions to upload the package to PyPI
+(and, in the future, also to Conda). To create a release:
+
+1. Change the "master" header in ``CHANGELOG.rst`` to the release
+   version number (not starting with "v", e.g. "1.2.3") and create a
+   new, empty "master" header above. Commit these changes with the
+   message, 'Prepare for release of vVERSIONNUMBER'' e.g. 'Prepare for
```
Comment (znicholls) [inline docs/development.rst]:
```suggestion
   message, "Prepare for release of vVERSIONNUMBER'' e.g. "Prepare for
```

File: docs/development.rst [hunk 3]
```diff
@@ -230,6 +230,25 @@ there).
...
+   release of v1.2.3'.
```
Comment (znicholls) [inline docs/development.rst]:
```suggestion
   release of v1.2.3".
```

**Comment by swillner** [comment]
> We shouldn't be able to commit directly to master at all?

I think we are stil allowed as administrators ;) But I could enforce that anyway ;) Ok?

**Comment by znicholls** [comment]
> But I could enforce that anyway ;) Ok?

That would be my strong preference

**Pull request merged**